.github/settings.yml

```diff
@@ -22,7 +22,7 @@ repository:
 
   # Uncomment this description property
   # and update the description to the current repo description.
-  # description: ""
+  description: "Terraform module to create and work with IBM watson Governance."
 
   # Use a comma-separated list of topics to set on the repo (ensure not to use any caps in the topic string).
-  topics: terraform, ibm-cloud, terraform-module, core-team
+  topics: terraform, ibm-cloud, terraform-module, core-team, watsonx, watson, governance, watson-governance, classifier, cognitive-search
```

Comment:
I am not sure if `classifier`, `cognitive-search` can be a part of governance as these are more related to `watson discovery` but `aiopenscale` can be added here.